### ISSUE TITLE
"Microsoft.Maui.Controls.Platform.ShellFlyoutItemView" is reported to screen readers instead of the shown text - poor accessibility

### ISSUE THREAD

**Issue by towebo** [dotnet/maui#12919]
### Description

The items in a flyout are spoken as their class, "Microsoft.Maui.Controls.Platform.ShellFlyoutItemView", instead of the text that's shown. This makes it impossible to choose the correct item when using a screen reader like NVDA or Narrator.

### Steps to Reproduce

1. Create a new standard MAUI application based on some template.
2. Change the flyout behavior to Shell.FlyoutBehavior="Flyout".
3. Run the application.
4. Start screen reader, Ctrl + Windows + Enter for Narrator.
5. Press Tab until you reach the open flyout button and press Enter to open it.
6. Press Tab to focus the first flyout item and listen to what's spoken.

Expected behavior
The same text that's rendered on screen should be spoken.

This type of bug is widely present in newer frameworks since UWP and can be seen in parts of Windows.

### Link to public reproduction project repository

https://github.com/towebo/MAUI-Flyout-Item-View-Issue

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 10 SDK (10.0.19041.0)

### Did you find any workaround?

No. I tried to override ToString() on the page but that didn't help. I can help in other situations on other classes is my experience.

**Comment by towebo** [comment]
Updated to Visual Studio 17.5 Preview 4 and that resolved the issue.
